src/lib/utilities.js

```diff
@@ -0,0 +1,12 @@
+/** Given some props, return only the props that start with "data-". */
+export default function getDataProps(props) {
+    return Object.keys(props).reduce((prev, key) => {
+        if (key.substr(0, 5) === 'data-') {
```

Comment:
Other libraries also white list 'aria-' props. But I'm keeping the scope to data attributes for now. Plus I don't know if it's appropriate to have aria attributes since I don't know much about accessibility.